MushroomObserver/mushroom-observer
# Remove thumbnail map

To test:
- On master:
-- Go to the show observation page and see the Map section on the right side of the screen
-- Go to the user preferences page and see an option for toggling whether map thumbnails are visible
-- See current banner

- On branch:
-- Go to the show observation page and see no Map section
-- Go to the user preferences page and see no option for map thumbnails
-- See banner with comment about the change
